sympy/physics/quantum/tests/test_spin.py

```diff
@@ -3699,84 +3699,83 @@ def test_jplus():
     # Normal operators, normal states
     # Numerical
     assert qapply(Jplus*JxKet(1, 1)) == \
-        -hbar*sqrt(2)*JxKet(1, 0)/2 + hbar*JxKet(1, 1)
+        -hbar*sqrt(2)*JxKet(1, 0)/2 + hbar*JxKet(1, 1) #2 no .expand()
     assert qapply(Jplus*JyKet(1, 1)) == \
-        hbar*sqrt(2)*JyKet(1, 0)/2 + I*hbar*JyKet(1, 1)
+        hbar*sqrt(2)*JyKet(1, 0)/2 + I*hbar*JyKet(1, 1) #2 no .expand()
```

Comment:
I don't think all these comments are needed.

Reply:
These particular comments are just for reference what i) has been done with the original test cases to make them pass with the new qapply, and ii) what modification is no longer necessary since qapply does moderate expand.  I believe we should keep i)  in any case. Remarks of type ii) (marked as #2) might be removed.